webrtc-stats.html

```diff
@@ -924,7 +926,8 @@
               </dt>
               <dd>
                 <p>
-                  Represents the number of unique datachannels opened.
+                  Represents the number of unique DataChannels returned from a successful
+                  createDataChannel() call.
```

Comment:
We may have to do that - which means that "opened" is "number of datachannels that have transitioned to the open state", which makes equal sense, but is a different number.